data/net/minecraft/client/renderer/debug/BeeDebugRenderer.mapping

```diff
@@ -1,7 +1,107 @@
 CLASS net/minecraft/client/renderer/debug/BeeDebugRenderer
-	METHOD render (Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;DDD)V
-		ARG 1 matrixStack
-		ARG 2 buffer
-		ARG 3 camX
-		ARG 5 camY
-		ARG 7 camZ
+	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
+		ARG 1 minecraft
+	METHOD addOrUpdateBeeInfo (Lnet/minecraft/client/renderer/debug/BeeDebugRenderer$BeeInfo;)V
+		ARG 1 beeInfo
+	METHOD addOrUpdateHiveInfo (Lnet/minecraft/client/renderer/debug/BeeDebugRenderer$HiveInfo;)V
+		ARG 1 hive
```

Comment:
There was some disparity between these in the method names so I tried to match the parent method of each parameter. (that said, this specific mapping doesn't follow that, I guess I missed it whilst reviewing.)

Anyway you're right, I think it's better to just choose a convention for them. I'll probably go with `xInfo` since the object doesn't represent `x` but simply holds information about it.